kriswallsmith/buzz/0.17/config/packages/buzz.yaml

```diff
@@ -0,0 +1,27 @@
+# Read more about client options at: https://github.com/kriswallsmith/Buzz/blob/master/doc/client.md
+services:
```

Comment:
the file should have defaults to make services private

Reply:
Fixed